lib/fog/storage/openstack.rb

```diff
@@ -43,7 +43,22 @@ class OpenStack < Fog::Service
       request :post_set_meta_temp_url_key
       request :public_url
 
+      module Utils
+        def require_mime_types
+          # Use mime/types/columnar if available, for reduced memory usage
+          require 'mime/types/columnar'
+          rescue LoadError
+            begin
+              require 'mime/types'
+            rescue LoadError
+              Fog::Logger.warning("'mime-types' missing, please install and try again.")
+              exit(1)
+            end
+          end
```

Comment:
I believe the issue is that when using implicit `begin` then `rescue` should be aligned with `def`. For instance:
```ruby
module M
  def foo
    ...
  rescue
    begin
    rescue
    end
  end
end
```